cpu/em_cpuhotplug.py

```diff
@@ -37,6 +37,7 @@ def setUp(self):
         if distro.detect().arch not in ['ppc64', 'ppc64le']:
             self.cancel("Only supported in powerpc system")
 
+        self.loop = self.params.get('test_loop', default=100)
```

Comment:
int(self.params.get('test_loop', default=100))

Reply:
ok sure makes sense if user had string in yaml